report/styles/report.css

```diff
@@ -43,18 +43,22 @@ html, body {
   background: #FAFAFA;
   color: #444;
   font-family: Arial, sans-serif;
-  font-size: 16px;
+  font-size: 15px;
 }
 
 body {
-  min-width: 600px;
+  min-width: 640px;
   padding: 0 16px;
 }
 
 a {
   color: #57A0A8;
 }
 
+.hidden {
```

Comment:
are you still using this, then?

Or would you want to add `*[hidden] { display: none !important; }` for < IE11?